x-pack/test/observability_functional/apps/observability/alerts/pagination.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+const ROWS_COUNT_TO_HIDE_PAGE_SELECTOR = 10;
+const DEFAULT_ROWS_PER_PAGE = 50;
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+
+  describe('Observability alerts pagination', function () {
+    this.tags('includeFirefox');
+
+    const retry = getService('retry');
+    const observability = getService('observability');
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
+      await observability.alerts.common.navigateToTimeWithData();
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+    });
+
+    describe(`When less than ${ROWS_COUNT_TO_HIDE_PAGE_SELECTOR} alerts are visible in the screen`, () => {
```

Comment:
I find "are visible in the screen" a little misleading. Maybe we could re-phrase this to "match the current query"? 
Meaning, when less than 10 alerts are found (with the current query and data), then we don't show any pagination.